drupal/web/modules/custom/wunder_next/wunder_next.module

```diff
@@ -52,3 +54,19 @@ function wunder_next_ckeditor5_plugin_info_alter(array &$plugin_definitions): vo
     $plugin_definitions['ckeditor5_imageResize'] = new CKEditor5PluginDefinition($image_resize_plugin_definition);
   };
 }
+
+/**
+ * Implements hook_simple_oauth_private_claims_alter().
+ */
+function wunder_next_simple_oauth_private_claims_alter(&$private_claims, AccessTokenEntity $access_token_entity) {
+  // Get the user id:
+  $user_id = $access_token_entity->getUserIdentifier();
+  $user = User::load($user_id);
+
+  // Add the email and the username to the claims.
```

Comment:
This hook allows us to add more info to the user information that is available to the frontend.